src/wp-admin/includes/ajax-actions.php

```diff
@@ -4588,6 +4588,32 @@ function wp_ajax_activate_plugin() {
 		wp_send_json_error( $status );
 	}
 
+	/**
+	 * Filters the data for the plugin's configuration workflow.
```

Comment:
```suggestion
	 * Filters the data for the plugin's configuration workflow.
	 *
	 * The dynamic portion of the hook refers to the plugin's slug provided by the
	 * AJAX activation request. For the Hello Dolly plugin, this would be `hello-dolly`.
```

Reply:
Nice. I like that. Thanks @peterwilsoncc